### PR TITLE
fix: correct display of the error in case of change `authenticatorHeader`

### PR DESCRIPTION
**Pull request by datamweb**
```php    
public array $authenticatorHeader = [
        'tokens' => 'MyToken',
    ];
```
**Befor:**
Every request must have a bearer token in the **Authorization** header.

**After:**
Every request must have a bearer token in the **MyToken** header.

### CODE REVIEW

**Review by kenjis — APPROVED** [review]

**Review by MGatner — APPROVED** [review]

**Comment by datamweb** [comment]
Add a note in the documentation and change the test commit message.

File: docs/guides/mobile_apps.md [hunk 1]
```diff
@@ -58,3 +58,9 @@ class LoginController extends BaseController
 ```
 
 When making all future requests to the API, the mobile client should return the raw token in the `Authorization` header as a `Bearer` token.
+
+> **Note**
+>
+> By default, `$authenticatorHeader['tokens']` is set to `Authorization`. You can change this value by setting the `$authenticatorHeader['tokens']` value in the `Auth.php` config file.
```
Comment (MGatner) [inline docs/guides/mobile_apps.md:64]:
```suggestion
> By default, `$authenticatorHeader['tokens']` is set to `Authorization`. You can change the header name by setting the `$authenticatorHeader['tokens']` value in the `Auth.php` config file.
```

File: docs/guides/mobile_apps.md [hunk 2]
```diff
@@ -58,3 +58,9 @@ class LoginController extends BaseController
...
+>
+> e.g. if `$authenticatorHeader['tokens']` is set to `PersonalAccessCodes`. the mobile client should return the raw token in the `PersonalAccessCodes` header as a `Bearer` token.
```
Comment (MGatner) [inline docs/guides/mobile_apps.md:66]:
```suggestion
> e.g. if `$authenticatorHeader['tokens']` is set to `PersonalAccessCodes` then the mobile client should return the raw token in the `PersonalAccessCodes` header as a `Bearer` token.
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Looks good! Small proofreading

**Review by kenjis — APPROVED** [review]

**Review by MGatner — APPROVED** [review]

**Pull request merged**